src/lib/api/product.ts

```diff
@@ -0,0 +1,13 @@
+import apiClient from './apiClient';
+
+interface GetProductsProps {
+  page?: number;
+  pageSize?: number;
+  tag_id?: number;
+  category_id?: number;
+  brand_id?: number;
+  select_content?: any;
+}
```

Comment:
```suggestion
interface IGetProductsProps {
  page?: number;
  pageSize?: number;
  tag_id?: number;
  category_id?: number;
  brand_id?: number;
  select_content?: any;
}
```

[quicktype](https://app.quicktype.io/)라는 툴이 있는데, JSON -> TS Interface로 만들어줘
나중엔 이걸로 해봐도 될듯?!